documentation/modules/deploying/proc-deploy-access-operator.adoc

```diff
@@ -0,0 +1,43 @@
+// This assembly is included in the following assemblies:
+//
+// security/assembly-access-operator.adoc
+
+[id='proc-deploy-access-operator-{context}']
+= Deploying the Access Operator (early access)
+
+[role="_abstract"]
+This procedure shows how to deploy the Access Operator for access management of a Kafka cluster.
+You cannot use the Access Operator with a Kafka cluster that is not managed by the Cluster Operator.  
+
+.Prerequisites
+
+* Installation requires a Kubernetes user with `cluster-admin` role, such as `system:admin`.
```

Comment:
It does not and saying something like this is normally very scary for some users. It requires users with the same privileges as the Access operator requires. I think I would simply leave this out.